patches/minecraft/net/minecraft/entity/passive/HorseArmorType.java.patch

```diff
@@ -0,0 +1,62 @@
+--- ../src-base/minecraft/net/minecraft/entity/passive/HorseArmorType.java
++++ ../src-work/minecraft/net/minecraft/entity/passive/HorseArmorType.java
+@@ -25,6 +25,14 @@
+         this.field_188587_f = "";
+     }
+ 
++    //Allows for textures located outside the vanilla horse armor folder
++    private  HorseArmorType(String textureLocation, String hash, int armorStrengthIn) 
```

Comment:
It can be package private based on the fact that only enum constants can use the constructor without reflection or asm